py/desitarget/cuts.py

```diff
@@ -1062,6 +1069,18 @@ def isBGS_colors(gflux=None, rflux=None, zflux=None, w1flux=None, w2flux=None,
         bgs &= rflux < gflux * 10**(4.0/2.5)
         bgs &= zflux > rflux * 10**(-1.0/2.5)
         bgs &= zflux < rflux * 10**(4.0/2.5)
+        
+    g = 22.5 - 2.5*np.log10(gflux.clip(1e-16))
+    r = 22.5 - 2.5*np.log10(rflux.clip(1e-16))
+    z = 22.5 - 2.5*np.log10(zflux.clip(1e-16))
+    rfib = 22.5 - 2.5*np.log10(rfiberflux.clip(1e-16))
```

Comment:
This change broke integration tests (currently debugging in https://github.com/desihub/desisim/pull/514) because we're not currently passing mock fiberfluxes to the targeting functions, and we don't test here against the possibility that `fiberflux is None` (the default input).

Reply:
This should now be addressed in #563.